.travis.yml

```diff
@@ -11,6 +11,12 @@ services:
   - mysql
 
 addons:
+  sonarcloud:
+    organization: "de-htwsaar-sqe-bildergalerie"
+    token:
+      secure: "gIE94T0VO9Uo+RPe1/bfd4+9nj/XVSo0XtiyNqeCLmb2AOJRaowLiqhZnBZ8bqgn5j/DFVHH35a69rrzbOQroxaPqrG70S9OOsDnNspVy7WjTMPrgl/MPCa6E3GKQXPRToI7x5BQeBKT3VFp5dlzolrp7DlbLhv8zR2PiHfxTmsVhOLW39eJup+MwCFWWCpL7VfBR/80fvqdZdE7a+HiKO6ysGKpndBkVT0Sgbfpwra6yeEnmnEWSm6FC4NaW55vizizKxwSAXXmmdO32xRLQjovNwkdIgYd1/4qdfUtlwZlw636+ROILmBfjkzqrVv3KxNSWUGh9sdmZZEEtA6GgwPdb+DyrAV3drulF1b4FVbINY0kgzM5bHMBE7Fs+vmLY0zlC5xATu8NiTi2SC9uF6VTXfAzMeHpaRNNEyjaqw0uhEppIEriDrN5/CCkJ3omg834BCWdZAk3hbkVfQLeAv9E0OCCLzGAGGqQFqoIf+FoQ401aTB9/M3GlGUMD74oF/Fc0otnSzeO8oX+kzjJAIuFrGXDlI6JlZcSdBnkn1RNtAErc95ra5/DgYMSkTj4LGIY4Io4xbt3Yw4eKgcE+DbbTW+5iG3cg/HU0cHHIne50VKUHNaXOmzmuCkL25jKJDFg6GqWL0PZAjNs2Yh3y/SDwOvzRy43MhE9dIHtdFo="
```

Comment:
I am not sure if it is a good idea to put a secure token under version control in a public repository 🤔 ? What is the purpose of this token?

On travis one can set environment variables: https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings. Maybe we should use this instead of setting the token in the `.travis.yml` ?

Reply:
The token is an [encrypted](https://docs.travis-ci.com/user/encryption-keys/#Encryption-scheme) version of the sonarcloud-token, so it should be safe to place it it here. Its possible to use a environement variable (`SONAR_TOKEN`) instead too. [Take a look here, Step 2](https://docs.travis-ci.com/user/sonarcloud/)

Reply:
Okay, we can stick to the encrypted version. Can we inspect our pull requests, too? Or do we have the same issues as with our feature branches? Should we rethink our branch name convention?

Reply:
Ok. [Is possible](https://docs.travis-ci.com/user/sonarcloud/#Activation-for-internal-pull-requests), but i think there would be the same issue. 
So I think its a good idea to change our naming convention for branches from `feature/#XX/readable-name` to only `feature/readable-name`. I do not see any problems there, because we always have only a few branches working on and they are not long living.

Reply:
Ok. Let's discuss our branching name convention in out next team meeting on tuesday. I will merge in ;-)